jsx/Filter.js

```diff
@@ -109,6 +101,38 @@ class Filter extends Component {
   }
 
   render() {
+    const filterPresets = () => {
+      if (this.props.filterPresets) {
+        const presets = this.props.filterPresets.map((preset) => {
+          const handleClick = () => this.props.updateFilter(preset.filter);
+          return <li><a onClick={handleClick}>{preset.label}</a></li>;
+        });
+        return (
+          <li className='dropdown'>
+            <a
+              className='dropdown-toggle'
+              data-toggle='dropdown'
+              role='button'
+            >
+              Load Filter Preset <span className='caret'/>
+            </a>
+            <ul className='dropdown-menu' role='menu'>
+              {presets}
+            </ul>
+          </li>
+        );
+      };
+    };
+
+    const filterActions = (
+      <ul className='nav nav-tabs navbar-right' style={{borderBottom: 'none'}}>
+        {filterPresets()}
+        <li>
+          <a role='button' onClick={this.props.clearFilter}>Clear Filter</a>
```

Comment:
@HenriRabalais Could u add a name = 'reset' inside this link element. It will be convient to do automation test.